cmd/ooniprobe/internal/cli/list/list.go

```diff
@@ -92,14 +94,17 @@ func init() {
 			netCount := make(map[uint]int)
 			output.SectionTitle("Results")
 			for idx, result := range doneResults {
-				totalCount, anmlyCount, err := database.GetMeasurementCounts(probeCLI.DB(), result.Result.ID)
-				if err != nil {
-					log.WithError(err).Error("failed to list measurement counts")
-				}
-				testKeys, err := database.GetResultTestKeys(probeCLI.DB(), result.Result.ID)
-				if err != nil {
-					log.WithError(err).Error("failed to get testKeys")
+				testKeys := "{}"
+
+				// We only care to expose in the testKeys the value of the ndt test result
```

Comment:
I would suggest to document the way in which we're merging the test keys using `|` and why we're doing that in a more thorough way. Because this is potentially technical debt, it's quite important that we ensure that we understand what are its boundaries, should we change that in the future. To this end I suggest to:

1. expand the comment above to explain what is going on (like why we're collapsing the keys)
2. make sure we mention the same in the place where we are merging the test keys

Reply:
I have pushed a commit that enriches these comments.